### PR TITLE
Hotfix for spirit-checking

### PR DESCRIPTION
**Pull request by wiredearp**
@nealstewart

### CODE REVIEW

File: src/gui-spirits@wunderbyte.com/mutations/gui.DOMCombos.js [hunk 1]
```diff
@@ -35,7 +35,8 @@ gui.DOMCombos = (function using(
 	 * @returns {boolean}
 	 */
 	var ifSpirit = provided(function() {
-		return Type.isSpirit(this) && !this.spirit.$destructed;
```
Comment (nealstewart) [inline src/gui-spirits@wunderbyte.com/mutations/gui.DOMCombos.js]: Can you fix Type.isSpirit instead?

Comment (wiredearp) [inline src/gui-spirits@wunderbyte.com/mutations/gui.DOMCombos.js]: 'this' is an element in this context, so Type.isSpirit correctly returns false.

Comment (nealstewart) [inline src/gui-spirits@wunderbyte.com/mutations/gui.DOMCombos.js]: ok... i guess the question is... "hasSpirit"

**Comment by nealstewart** [comment]
Test?

File: src/gui-spirits@wunderbyte.com/mutations/gui.DOMCombos.js (hunk 1 above)
Comment (wiredearp) [inline src/gui-spirits@wunderbyte.com/mutations/gui.DOMCombos.js]: Right, that's however not for the type-checker to know.

But this is also what the code does now.

File: dist/spiritual-gui-0.2.3.js [hunk 2]
```diff
@@ -5625,6 +5643,8 @@ gui.Client = (function() {
 		// user agent death match - this obviously needs some work
 		this.isExplorer = agent.contains("msie") || agent.contains("trident");
 		this.isExplorer9 = this.isExplorer && agent.contains("msie 9");
+		this.isExplorer10 = this.isExplorer && agent.contains("msie 10");
```
Comment (nealstewart) [inline dist/spiritual-gui-0.2.3.js]: OooOoo

File: dist/spiritual-gui-0.2.3.js [hunk 3]
```diff
@@ -9974,9 +9994,9 @@ gui.Spirit = gui.Class.create(Object.prototype, {
 	ondetach: function() {},
 
 	/**
-	 * `onexit` gets if the spirit element has been *manually* detached 
-	 * and not re-attached in the same execution stack. Spirit is not 
-	 * positioned in the document DOM at this point.
+	 * `onexit` gets called if the spirit element has been *manually* detached
```
Comment (nealstewart) [inline dist/spiritual-gui-0.2.3.js]: it'd be nice to have an overview of all these events in a file somewhere.

Comment (nealstewart) [inline dist/spiritual-gui-0.2.3.js]: A cute little ascii art thing of things that occur and events that are triggered. Woo

**Comment by nealstewart** [comment]
:+1:

File: dist/spiritual-gui-0.2.3.js (hunk 3 above)
Comment (wiredearp) [inline dist/spiritual-gui-0.2.3.js]: For now there's http://wunderbyte.github.io/spiritual-gui/lifecycle.html.

**Pull request merged**